src/client/BuilderClient.ts

```diff
@@ -24,8 +25,10 @@ export class BuilderClient {
     this.getAddress = () => (address instanceof Function ? address() : address)
 
     this.fetch = (path: string, init?: RequestInit) => {
-      const method: string = init?.method ?? path ?? 'get'
-      const fullUrl = url + path
+      const URL = globalThis.URL ?? nodeUrl.URL
```

Comment:
Does the globalThis exist on node? 
If not, it will break when trying to access .URL